src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -36,7 +36,11 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
     "artwork-page-create-alert"
   )
 
+  // If we have info about the lot end time (cascading), use that.
   const { sale, saleArtwork } = artwork
+  const endAt = saleArtwork?.endAt
+  const startAt = sale?.startAt
+  const { hasEnded } = useTimer(endAt!, startAt!)
```

Comment:
If there's no `endAt` on the `saleArtwork` (i.e. no cascade end times) will the hook just return `null` for `hasEnded`?

Reply:
Great question!

Exactly, it will return `false`, so it works as we want. I was looking into making the hook behave more appropriately when those values were missing, so it could be reused here, and I was surprised that it worked already!

Reply:
Oh that's great! so `hasEnded` at here means "the lot has ended or it's not cascading end times." love it